packages/framework/src/constants/workflow-options.constants.ts

```diff
@@ -0,0 +1,8 @@
+export enum WorkflowOptionsChannelEnum {
```

Comment:
Can we use `ChannelTypeEnum` for the values that are the same?

Reply:
Enums can't refer to other enum values (by using computed values), so it would either be this enum with a single value or the way it is here -- with all keys in the framework setting using this new enum

Reply:
I actually prefer to keep them separate as they are now to make the distinction between the two more clear - one is machine readable, the other is human readable.